bin/permadelete_crash_data.py

```diff
@@ -108,18 +110,18 @@ def s3_delete(crashid):
     bucket = s3_context.config.bucket_name
     s3_client = s3_context.client
 
-    # Delete the raw crash
-    key = "v2/raw_crash/%(entropy)s/%(date)s/%(crashid)s" % {
-        "entropy": crashid[0:3],
-        "date": "20" + crashid[-6:],
-        "crashid": crashid,
-    }
-    obj = s3_fetch_object(s3_client, bucket, key)
-    if obj:
-        s3_delete_object(s3_client, bucket, key)
+    keys = build_keys("raw_crash", crashid)
+    for key in keys:
+        try:
+            obj = s3_fetch_object(s3_client, bucket, key)
+            if obj:
+                s3_delete_object(s3_client, bucket, key)
```

Comment:
In this case, while it's very unlikely we could have the data at more than one key, it's possible, so we should iterate through all the keys and not do a `break`.